### DISCUSSION TITLE
How can I use firebase emulator with the module

### DISCUSSION THREAD

**Discussion by USER4247** [AsifArmanRahman/firebase-rest-api#14] [Q&A]
I have a kivy app . I want to test the app using emulator . How can I use emulatorin this module ? . Please provide an example . Thanks

**Comment by Lxstr** [comment]
Have you read the documentation at [[firebase-rest-api.readthedocs.io](https://firebase-rest-api.readthedocs.io/)]([firebase-rest-api.readthedocs.io](https://firebase-rest-api.readthedocs.io/))? What is not working for your use case?
